### PR TITLE
Update clone-bundle route

### PR DESCRIPTION
**Pull request by yixu34**

### CODE REVIEW

File: launch/client.py [hunk 1]
```diff
@@ -608,7 +611,7 @@ def create_model_bundle(  # pylint: disable=too-many-statements
                 schema_location=schema_location,
             )
             create_model_bundle_request = CreateModelBundleRequest(**payload)  # type: ignore
-            api_instance.create_model_bundle_v1_model_bundles_post(
+            resp = api_instance.create_model_bundle_v1_model_bundles_post(
```
Comment (phil-scale) [inline launch/client.py]: Unused variable might throw lint error?

**Review by phil-scale — APPROVED** [review]
Something feels unergonomic about needing to get the model bundle ID before cloning but I think this API makes sense. Can probably just build something on top

**Comment by yixu34** [comment]
> Something feels unergonomic about needing to get the model bundle ID before cloning but I think this API makes sense. Can probably just build something on top

Oh I see, like we should also accept a `ModelBundle`. Let me see.

File: launch/client.py (hunk 1 above)
Comment (yixu34) [reply launch/client.py]: Oh yeah I removed this.

**Pull request merged**